_posts/fr/2023-05-17-policy.md

```diff
@@ -45,12 +45,18 @@ h2:not(:first-of-type) { margin-top: 3em; }
 
 {% include specials/policy/fr/03-encheres-pour-l-achat-d-un-bloc-d-espace.md %}
 
-# Estimation des taux de frais
+## Estimation des taux de frais
```

Comment:
Added a correction here which caused misformatting of the bullets on the page

Reply:
So Maybe it's wrong for past news

Reply:
I didn’t catch it before. I dont think it affected the formatting (or I didn’t notice) until an additional entry for the series was added.

Reply:
Complicated and confuse for me

Reply:
Two changes in https://github.com/bitcoinops/bitcoinops.github.io/pull/1155/commits/a1f5aaaada7870664fd1686c4e849e102006cb4e

- Make the headers consistent in that we have used ## for the other headings
- Take out the text 'En attente de confirmation #5 : ' text since we dont include that text on the policy page